tests/settings.py

```diff
@@ -2,16 +2,18 @@
 from __future__ import unicode_literals, absolute_import
 
 import django
-import dj_database_url
 
 DEBUG = True
 USE_TZ = True
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "hhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh"
+SECRET_KEY = "uzbLoOIYlJnzGDYlUfynNyocjZH9NLSc3AAREwLDaugQkCzsQn"
 
 DATABASES = {
-    "default": dj_database_url.config(default='sqlite://:memory:')
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
```

Comment:
this is probably also a left-over from sqlite testing, right? This basically disables testing for mysql and postgresql.

Reply:
I want to remove the dependency to `dj_database_url` which is unnecessary. Thats why I wrote it this way. Ok for you?

Reply:
it's ok in principle, but you'd have to find a comparably easy way of [loading these settings from the environment](https://github.com/RegioHelden/django-scrubber/blob/master/tox.ini#L10-L12).
I'd suggest we leave it to a different PR and focus on the feature here.

Reply:
But it works now and we need one package less. Whats the drawback? 🤔

Reply:
It "works" because all tests are using sqlite everywhere, even those that should be using mysql or postgres. We're not testing what we think we're testing.

Reply:
And with your line, it is different?

Reply:
Yeah, the tox.ini defines the database urls for the dj_database_url plugin as environment variables and apparently the plugin has predefined environment variables where it'll look for the config without explicitly reading the environment variables into the settings.py

Also I'm with @costela on this: Make single-purpose PRs and just create a new one if you'd like to suggest a different style of things. It's just a stylistic choice and there's room for arguing about that on a subjective basis, while improving the faker thing is an objective improvement which everyone seems to have agreed on already.